README.md

```diff
@@ -159,21 +159,19 @@ information.
 
 # Testing
 There are currently three levels of tests: unit tests, component tests, and
-service tests. To run all tests after building, run `run-all-tests.sh`
-in the project root.
+service tests.
 
 The unit tests written in gtest/gmock are available in */unit-test/*. The tests
-are run using `run-tests.sh` from the build directory. Note that you have
+are run using `run-tests.py` from the build directory. Note that you have
 to run them as root, since the actual container creation is not stubbed of
```

Comment:
Please take the opportunity to fix the "of" typo. It should be "off", or not included at all.